Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -82,6 +82,9 @@ def run
           We'd still love your contributions, though, so try another one
           that is excluded from autobump list (i.e. it has 'no_autobump!'
           method or 'livecheck' block with 'skip'.)
+
+          More information about autobump process can be found here:
```

Comment:
@botantony On second thought: let's split this PR into two:
1. the actual logic changes to use the new autobump system
2. the documentation changes

The two do not need to be in the same PR and it's only going to slow things down to have them both in here.

Reply:
I agree.